### PR TITLE
[SITE-2632] Alternative relay check

### PR DESCRIPTION
**Pull request by pwtyler**
Alternative version of #471 with bool adjustment, green tests, and changelog update.

### CODE REVIEW

File: object-cache.php [hunk 1]
```diff
@@ -1226,7 +1226,12 @@ protected function _connect_redis() {
 	 *               not with a message describing the issue.
 	 */
 	public function check_client_dependencies() {
-		if ( ! class_exists( 'Redis' ) ) {
+		$class_to_check = 'Redis';
```
Comment (jazzsequence) [inline object-cache.php:1229]: Maybe this should be a filter? 🤔

**Review by jazzsequence — APPROVED** [review]

File: object-cache.php (hunk 1 above)
Comment (pwtyler) [reply object-cache.php:1229]: Not right now.

**Comment by pwtyler** [comment]
@EarthlingDavey, do you have a WordPress.org profile you'd like us to include in readme.txt changelog?

**Review by greg-1-anderson — APPROVED** [review]

**Comment by EarthlingDavey** [comment]
Hey @pwtyler thanks for the mention in the readme ... I don't have a .org profile.

**Pull request merged**